### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ZmijaWA**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/new-feature.md [hunk 1]
```diff
@@ -99,6 +99,13 @@ _Add test scenarios for the new feature_
 * [ ] Concent integration tests pass
 * [ ] Concent acceptance tests pass
 
+
+**Please choose the priority label for QA. It is set to the lowest by default. To setup higher priority please change the label**
+_P0 label is set for Severity-Critical/Effort-easy
```
Comment (shadeofblue) [inline .github/ISSUE_TEMPLATE/new-feature.md:104]: @ZmijaWA is this underscore character here intentional?

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ assignees: ZmijaWA
 
 **Mainnet/Testnet**:
 
+**Priority label is set to the lowest by default. To setup higher priority please change the label**
+_P0 label is set for Severity-Critical/Effort-easy
```
Comment (shadeofblue) [inline .github/ISSUE_TEMPLATE/bug-report.md:25]: @ZmijaWA is this underscore here intentional?

File: .github/ISSUE_TEMPLATE/new-feature.md (hunk 1 above)
Comment (ZmijaWA) [reply .github/ISSUE_TEMPLATE/new-feature.md:104]: @shadeofblue yes :)

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (ZmijaWA) [reply .github/ISSUE_TEMPLATE/bug-report.md:25]: @shadeofblue yes :)

**Review by shadeofblue — APPROVED** [review]

**Pull request merged**